compiler/passes/InitNormalize.cpp

```diff
@@ -1168,7 +1168,12 @@ void InitNormalize::updateFieldsMember(Expr* expr) const {
         SymExpr* _this = new SymExpr(mFn->_this);
         SymExpr* name  = new SymExpr(new_CStringSymbol(sym->name));
 
-        symExpr->replace(new CallExpr(PRIM_GET_MEMBER, _this, name));
+        if (field->sym->hasFlag(FLAG_PARAM) ||
+            field->sym->hasFlag(FLAG_TYPE_VARIABLE)) {
+          symExpr->replace(new CallExpr(PRIM_GET_MEMBER_VALUE, _this, name));
```

Comment:
Man, we seem to do this a lot.  I wonder if there's a better way to go about it?

Definitely fine in this PR, though